roles/content/tasks/1_manifest.yml

```diff
@@ -0,0 +1,29 @@
+- name: manifest | pull down manifest
```

Comment:
we have `theforeman.foreman.redhat_manifest` for that. would you want to switch to that now, or later?

Reply:
I will upload the new code later